src/Components/SimpleModal/SimpleModal.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { StyleSheet, TouchableOpacity, View } from "react-native";
+import Modal, { ModalProps } from "react-native-modal";
+import getShadowStyles from "src/Themes/getShadowStyle";
+import { isWeb } from "src/Helpers/platform";
+import Colors from "src/Themes/Colors";
+
+const documentBody = document.getElementsByTagName("body")[0];
+
+interface IProps {
+  isVisible: ModalProps["isVisible"];
```

Comment:
Why not to extend all ModalProps?

What will happen if you want to use some other prop's of `react-native` modal? In your case you should change the code here.

Yep, extending the complete object will lead to incompatibility issues in `web` environment, but this is a minimal implemenation with graceful degradation of the functionality and it should not support all the features.